doc/common/README.md

```diff
@@ -569,14 +569,51 @@ d. Участники, проверяющие код, добавляют ком
 
 e. Комментарии отмечает выполненными автор изменений, код которого просматирвают.
 
-f. После корректировки замечаний и реализации предложений должен запрашиваться
+f. В начале процесса исправления замечаний задача переводится в соотвествующий рабочий статус. 
+   После завершения исправлений задача переводится обратно в статус ревью.
+
+g. После корректировки замечаний и реализации предложений должен запрашиваться
 повторный обзор кода.
 
-g. Все вопросы по ревью обсуждаются наибыстрейшим из возможных способов. Недопустимо
+h. Все вопросы по ревью обсуждаются наибыстрейшим из возможных способов. Недопустимо
 использовать такой способ коммуникации, при котором ответ затягивается, а соответственно
 затягивается и ревью.
 
-#### 6.4.4. Противоречия и споры в процессе обзора кода
+#### 6.4.4. Процесс обзора задачи, не связанных с системой управления кодом (Github, Gitlab)
+
+Такие задачи, как документирование в Notion, Confluence и пр. 
+
+Мотивация:\
+Сократить количество отвлекающих воздействий на автора изменений до 2 (в позитивном сценарии), 
+меньше отвлекая друг друга от работы. 
+
+1. Рецензент в процессе обзора кода фиксирует свои замечания и пожелания в одном 
+   месте. \
+   Например: тематический канал в Slack, с отключёнными уведомлениями. \
+   Формат сообщений: \
+       1: [Номер задачи]: [краткое описание задачи], [ссылка на задачу] \
+       2: [Замечание или пожеление 1] \
+       3: [Замечание или пожеление 2] \
+       N+1: [Замечание или пожеление N]
+
+2. После того, как рецензент завершил фиксирование замечаний, он уведомляет об этом автора задач. 
+   А также переводит задачу в статус «открыта» / «переоткрыта», чтобы другие рецензенты делали 
+   обзоры только на непросмотренные задачи.
+
+3. Автор задачи знакомится со списком замечаний и пожеланий, называет список вопросов и обсуждает 
+   их с рецензентом. В начале работы над исправлениями по замечаниям автор задачи меняет статус задачи
+   на соответствующий рабочему.
+ 
+4. Автор задачи отмечает сделанные задачи смайлом: ✔ 
+ 
+5. После того, как автор задачи завершил улучшение проделанной работы и отметил все пункты как 
+   завершённые, он также сообщает об этом рецензенту.
```

Comment:
I suggest to not use indent for all lines of list item because it is difficult to maintain it. See list item below.

Reply:
Done